docs/dev/api/api-testing.md

```diff
@@ -2741,6 +3143,48 @@ Returns metrics for all tests in a project.
      <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>Your project's hook ID, which you can create and/or retrieve from your project's <strong>Webhooks</strong> tab.</p></td>
     </tr>
   </tbody>
+  <tbody>
+    <tr>
+     <td><code>from</code></td>
+     <td><p><small>| QUERY | OPTIONAL | INTEGER |</small></p><p>Identifies the start time, in UNIX time milliseconds. The default value is 2 days before.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+      <tr>
+      <td><code>to</code></td>
+      <td><p><small>| QUERY | OPTIONAL | INTEGER |</small></p><p>Identifies the end time, in UNIX time milliseconds. The default value is the current time.</p></td>
+      </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>limit</code></td>
+     <td><p><small>| QUERY | OPTIONAL | INTEGER |</small></p><p>Identifies the number of records to return. The default value is 100 and the max value is 500.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>offset</code></td>
+     <td><p><small>| QUERY | OPTIONAL | INTEGER |</small></p><p>Specifies the number of events to be skipped from the beginning of the list. Needs to be used in combination with the limit.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>failuresOnly</code></td>
+     <td><p><small>| QUERY | OPTIONAL | BOOLEAN |</small></p><p>Filters the metrics to include only the tests that completed with failure(s).</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>footprint</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>Filters the metrics based on the footprint.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>buildId</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>If this execution is associated with a build, the ID of the build should be provided so that all resources can be associated to it.</p></td>
```

Comment:
Not sure if "test run" is correct? 
```suggestion
     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>If the test run is associated with a build, provide the build ID so that all resources can be associated to it.</p></td>
```